repo2docker/buildpacks/base.py

```diff
@@ -463,7 +465,7 @@ def render(self):
             appendix=self.appendix,
         )
 
-    def build(self, client, image_spec, memory_limit, build_args, cache_from):
+    def build(self, client, image_spec, memory_limit, build_args, cache_from, extra_build_kwargs):
```

Comment:
```suggestion
    def build(self, client, image_spec, memory_limit, build_args, cache_from, extra_build_kwargs=None):
```
What do you think of adding this as a named parameter with a default value of `None`? This means that existing calls to `build()` will not need updating. I think this is more part of the internal API of repo2docker so we don't have to do this.